stable/awsebscsidriver/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.3.0"
+description: AWS EBS CSI Driver
+name: awsebscsidriver
```

Comment:
why are we adding awsebscidriver as a chart ? It is part of the awsebscsiprovisioner at this moment.

Reply:
To simplify the addons, we provide fout storage addons, localpath, localvolume, ebscsi and ebs.

Reply:
https://jira.mesosphere.com/browse/DCOS-56399

Reply:
This is super confusing, not to mention the naming we are using for all of this. Provisioner vs. Driver is still a valid change. Let's settle this in standup.

Reply:
That was the decision taken during the storage meetings. But, yeah, let's discuss it during the standup.